src/test/scala/uk/gov/hmrc/hmrcfrontend/views/config/HmrcPageHeadingSpec.scala

```diff
@@ -31,11 +32,11 @@ class HmrcPageHeadingSpec extends AnyWordSpec with Matchers {
         isPageHeading = true,
         content = HtmlContent(
           """What is your name? """
-            + """<span class="hmrc-caption">"""
+            + """<span class="hmrc-caption govuk-caption-xl">"""
             + """Personal details"""
             + """</span>"""
         ),
-        classes = "govuk-label--xl hmrc-page-heading govuk-!-margin-top-0 govuk-!-margin-bottom-2"
+        classes = "hmrc-page-heading govuk-!-margin-top-0 govuk-!-margin-bottom-2 govuk-label--xl"
```

Comment:
hopefully this reordering doesn't change the styles that get applied, I think it should be ok - I'm probably misremembering how css stuff applies and it doesn't matter

Reply:
I will do a manual test by using the HmrcPageHeaderLegend & HmrcPageHeaderLabel in contact frontend for this.

Reply:
Seems to be fine, the only thing I can spot is if we have a class like "govuk-fieldset__legend--m govuk-fieldset__legend--xx" the last CSS style will match the regex but xx is not a valid size?

![image](https://user-images.githubusercontent.com/1393788/162200439-54fb5f59-8600-4b5a-b5f4-b2015baff56a.png)

Reply:
if we can do that easily then that sounds like a useful extra bit of protection, I guess someone might create their own custom modifier maybe - but happy either way because I think that's something services can fix and verify themselves most likely